app/src/main/kotlin/com/wire/android/ui/calling/ongoing/OngoingCallScreen.kt

```diff
@@ -138,6 +146,12 @@ private fun OngoingCallContent(
     val scaffoldState = rememberBottomSheetScaffoldState(
         bottomSheetState = sheetState
     )
+
+    var shouldFullScreen by remember { mutableStateOf(false) }
+    var userId by remember { mutableStateOf(UserId(String.EMPTY, String.EMPTY)) }
+    var isSelfUser by remember { mutableStateOf(false) }
+    var clientId by remember { mutableStateOf(String.EMPTY) }
```

Comment:
Well, `OngoingCallScreen` will be a large a file if we do everything there, we still have to do zooming and display some toast indicator. 
Also we need to keep observing user state when we are on fullscreen, passing `UiParticipant` only will make it static. But yes I should rename them to better name

Reply:
I mean, it's already added to the `OngoingCallScreen` so my proposition was to simplify it actually 😅 
I thought that instead of four state vars:
```
    var shouldFullScreen by remember { mutableStateOf(false) }
    var userId by remember { mutableStateOf(UserId(String.EMPTY, String.EMPTY)) }
    var isSelfUser by remember { mutableStateOf(false) }
    var clientId by remember { mutableStateOf(String.EMPTY) }
```
 we could have only one which should work the same:
```
    var fullScreenParticipant: UICallParticipant? by remember { mutableStateOf(null) }
```

And by getting rid of `FullScreenTile` I didn't meant to move everything into `OngoingCallScreen` but to just use `ParticipantTile` directly 😉

Reply:
How is it not related to the participant if it's the particular participant that's shown on the full screen, right? Or can anything else other than the participant also be shown on the full screen?